packages/medusa/src/loaders/module.ts

```diff
@@ -0,0 +1,41 @@
+import { asValue } from "awilix"
+import { ConfigModule, Logger, MedusaContainer } from "../types/global"
+
+type Options = {
+  container: MedusaContainer
+  configModule: ConfigModule
+  logger: Logger
+}
+
+export default async ({
+  container,
+  configModule,
+  logger,
+}: Options): Promise<void> => {
+  const moduleResolutions = configModule?.moduleResolutions ?? {}
+  for (const [_, resolution] of Object.entries(moduleResolutions)) {
+    if (resolution.shouldResolve) {
+      try {
+        const loadedModule = await import(resolution.resolutionPath!)
+        console.log(resolution)
+        const moduleLoaders = loadedModule.loaders
+        if (moduleLoaders) {
+          await Promise.all(
```

Comment:
suggestion: Promise.all does not do anything since there is no I/O operation, they will be pushed to the event loop stack and resolved sequentially. a for...of should be enough